webhook-notifications_5def901604286364bc92af37.md

```diff
@@ -0,0 +1,283 @@
+Semaphore has integrated Webhook based notifications that are delivered on
+the success or failure of a pipeline.
+
+- [Setting up Webhook notification for a project](#setting-up-webhook-notifications-for-a-project)
```

Comment:
Use "webhook" (lower case) in the document. We use uppercase for brand names like Slack, Docker, Github, but not for general concepts like webhooks.